### ISSUE TITLE
Misc FE improvements

### ISSUE THREAD

**Issue by leonardoarcari** [leonardoarcari/dcapal#35]
### Action items

- Sunset `ImageKit` (which is penalizing Lighthouse score and makes asset management less agile)
- Replace `ImageKit` with Webpack's [Asset Modules](https://webpack.js.org/guides/asset-modules/#resource-assets)
- Fix text wrapping in `EndStep` in mobile view

<img width="254" alt="image" src="https://github.com/leonardoarcari/dcapal/assets/811969/b1d64c54-7fe4-4f28-8141-b6089298e22d">

#### Rust: fix workspace config

- Profiles for non root package

```
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   C:\Users\leonardoarcari\Development\rust\dcapal\dcapal-optimizer-wasm\Cargo.toml
workspace: C:\Users\leonardoarcari\Development\rust\dcapal\Cargo.toml
```

- Crates resolver v2

```
warning: some crates are on edition 2021 which defaults to `resolver = "2"`, but virtual workspaces default to `resolver = "1"`
note: to keep the current resolver, specify `workspace.resolver = "1"` in the workspace root's manifest
note: to use the edition 2021 resolver, specify `workspace.resolver = "2"` in the workspace root's manifest
```

- Move shared crate dependencies to workspace root ([link](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table))

**Comment by corradopetrelli** [comment]
For wrapping, I think it's sufficient to remove _whitespace-nowrap_ here:

https://github.com/leonardoarcari/dcapal/blob/44e3b6c0e465d4547f2da1edbaf0335cc81ffd9f/dcapal-frontend/src/components/allocationFlow/endStep/allocateCard.js#L103

### Demonstration
https://github.com/leonardoarcari/dcapal/assets/36475130/3587014a-e3a1-4518-bc4b-4c1485579f56

**Comment by corradopetrelli** [comment]
> Fix text wrapping in EndStep in mobile view


Resolved on #37

**Comment by leonardoarcari** [comment]
Resolved in #41
